vespa/package.py

```diff
@@ -1370,6 +1370,48 @@ def __repr__(self):
         )
 
 
+class ApplicationConfiguration(object):
+    def __init__(self, name: str, value: Union[str, Dict[str, Union[Dict, str]]]) -> str:
+        """
+        Create a Vespa Schema.
+
+        Check the `Config documentation <https://docs.vespa.ai/en/reference/services.html#generic-config>`__
+        for more detailed information about generic configuration.
+
+        :param name: Configuration name.
+        :param value: Either a string or a Dict (it may be a nested dict) of values.
+
+        Example:
+        >>> ApplicationConfiguration(
+        >>>     name="container.handler.observability.application-userdata",
+        >>>     value={"version": "my-version"}
+        >>> )
+        """
```

Comment:
Unit test fail is related to this. The doctesting fails here. Should be "..." on these lines?